DQM/L1TMonitor/src/L1TdeStage2uGT.cc

```diff
@@ -157,21 +164,35 @@ void L1TdeStage2uGT::analyze(const edm::Event& event, const edm::EventSetup& es)
           if (unprescaled) {
             if (it_data->getAlgoDecisionFinal(algoBit)) {
               hname = "DataNoEmul_" + bxt.str();
-              fillHist(m_SummaryHistograms, hsummary, float(NFinalMismatchDataNoEmul), 1.);
+              foundFinalMismatchDataNoEmul = true;
               wt = 1;
             } else {
               hname = "EmulatorNoData_" + bxt.str();
-              fillHist(m_SummaryHistograms, hsummary, float(NFinalMismatchEmulNoData), 1.);
+              foundFinalMismatchEmulNoData = true;
               wt = -1;
             }
             fillHist(m_HistNamesFinal, hname, float(algoBit), 1.);
-            finalDecisionMismatches_vs_LS->Fill(float(lumi), wt);
+            finalDecisionMismatches_vs_LS->Fill(
+                float(lumi),
+                wt);  // TODO: I think this weight makes things confusing. If we have uncorrelated no emu and a few no data mismatches it would look like we have fewer mismatches than we actually do.
```

Comment:
I spotted this (and the same line further below) while editing the file. If I understand correctly we're incrementing that histogram when we have a data, but no emulator mismatch and decrementing it when we have a emulator but no data mismatch. I think this doesn't make much sense, we should probably have two histograms in my opinion..

Reply:
I split the vs. LS histograms in two as proposed above.